spec/models/membership_application_spec.rb

```diff
@@ -30,6 +30,20 @@
     it { is_expected.to validate_length_of(:company_number).is_equal_to(10) }
   end
 
+  describe 'Validate Swedish Orgnr' do
+    let (:company) do
+      create(:membership_application)
+    end
+
+    subject { company }
+
+    before do
```

Comment:
I'm not clear on what this `before do...` block does.

And is there a test to see if a valid number is accepted?